pkg/controller/operators/olm/operatorgroup.go

```diff
@@ -293,7 +302,7 @@ func (a *Operator) ensureTenantRBAC(operatorNamespace, targetNamespace string, c
 
 func (a *Operator) copyCsvToTargetNamespace(csv *v1alpha1.ClusterServiceVersion, operatorGroup *v1alpha2.OperatorGroup) error {
 	namespaces := make([]string, 0)
-	if len(operatorGroup.Status.Namespaces) == 1 && operatorGroup.Status.Namespaces[0] == corev1.NamespaceAll {
+	if operatorGroup.Status.Namespaces[0] == corev1.NamespaceAll {
```

Comment:
It would be nice if we didn't have to implicitly rely on list ordering here

Reply:
I thought this was better than searching the string each time to see if "" was in the list. I couldn't figure out any naming issues with sorting not causing "" to be first.

Reply:
see above comments